dotCMS/src/main/java/com/dotcms/xmlsitemap/XMLSitemapJob.java

```diff
@@ -53,6 +50,20 @@
 import com.dotmarketing.util.UtilMethods;
 import com.dotmarketing.util.XMLUtils;
 import com.liferay.portal.model.User;
+import java.io.File;
+import java.io.InputStream;
+import java.io.OutputStreamWriter;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.GZIPOutputStream;
+import org.quartz.Job;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)